### DISCUSSION TITLE
RPC issues

### DISCUSSION THREAD

**Discussion by HarishankarVj12** [Fantom-foundation/go-opera#476] [General] [answered]
INFO [06-20|05:02:25.058] Loaded local transaction journal         transactions=0 dropped=0
INFO [06-20|05:02:25.058] Regenerated local transaction journal    transactions=0 accounts=0
INFO [06-20|05:02:25.062] Starting peer-to-peer node               instance=go-opera/v1.1.2-rc.3-e529a4e7-1669028682/linux-amd64/go1.20.5
INFO [06-20|05:02:25.447] New local node record                    seq=2 id=d2949e8924227460 ip=127.0.0.1 udp=5050 tcp=5050
INFO [06-20|05:02:25.801] Started P2P networking                   self=enode://be9220fe16719f7511b6f4c6ca14304dc81aaae78fc467cccc991d66bf64f9c90d71582ea9bcda56ad1e713c41eaf0f8607075390c9c774e96e79b799a7de9cb@127.0.0.1:5050
INFO [06-20|05:02:25.802] IPC endpoint opened                      url=/home/ubuntu/.opera/opera.ipc
INFO [06-20|05:02:25.803] EVM snapshot                             module=gossip-store at=e796b5..25cc43 generating=false
INFO [06-20|05:32:26.877] Looking for peers                        peercount=0 tried=5 static=0
INFO [06-20|06:02:25.059] Regenerated local transaction journal    transactions=0 accounts=0
INFO [06-20|06:02:25.942] Looking for peers                        peercount=0 tried=5 static=0

Any one helps out to solve this problem I am asking this question in Discord no one responding, I am facing this issue pass week.

**Comment by quan8** [comment]
Hi @HarishankarVj12 , are you using go1.20? Could you please try again with go1.19?
Which go-opera version are you using?

**Reply by HarishankarVj12** [reply]
INFO [06-21|17:39:42.684] Starting peer-to-peer node               instance=go-opera/v1.1.2-rc.3-e529a4e7-1669028682/linux-amd64/go1.19
INFO [06-21|17:39:42.702] New local node record                    seq=4 id=d2949e8924227460 ip=127.0.0.1 udp=5050 tcp=5050
INFO [06-21|17:39:42.704] Started P2P networking                   self=enode://be9220fe16719f7511b6f4c6ca14304dc81aaae78fc467cccc991d66bf64f9c90d71582ea9bcda56ad1e713c41eaf0f8607075390c9c774e96e79b799a7de9cb@127.0.0.1:5050
INFO [06-21|17:39:42.705] IPC endpoint opened                      url=/home/ubuntu/.opera/opera.ipc
INFO [06-21|17:39:42.706] EVM snapshot                             module=gossip-store at=e796b5..25cc43 generating=false
INFO [06-21|18:09:43.227] Looking for peers                        peercount=0 tried=5 static=0

I have changed go version but it is still showing same

**Comment by quan8** [comment]
Hi @HarishankarVj12 , which command did you use?
Can you take a look at this guide?
https://docs.fantom.foundation/node/run-a-read-only-node

**Reply by HarishankarVj12** [reply]
ding=7803
INFO [06-22|17:12:50.836] New LLR summary                          last_epoch=0 last_block=64527577 new_evs=0 new_ers=0 new_bvs=0   new_brs=239  age=10mo9d14h
INFO [06-22|17:12:59.970] State heal in progress                   accounts=0@0.00B slots=0@0.00B codes=55@554.23KiB nodes=1,393,885@492.69MiB pending=9889
INFO [06-22|17:13:13.955] State heal in progress                   accounts=0@0.00B slots=0@0.00B codes=55@554.23KiB nodes=1,394,626@493.07MiB pending=10767
INFO [06-22|17:13:13.974] New LLR summary                          last_epoch=0 last_block=64527577 new_evs=0 new_ers=0 new_bvs=0   new_brs=1571 age=10mo9d13h
INFO [06-22|17:13:22.002] New LLR summary                          last_epoch=218838 last_block=64527577 new_evs=6 new_ers=1 new_bvs=0   new_brs=2375 age=18.384s
INFO [06-22|17:13:30.082] New LLR summary                          last_epoch=218838 last_block=64527639 new_evs=0 new_ers=0 new_bvs=248 new_brs=3101 age=26.464s
I am getting logs in latest block But I execute this command <./opera attach --exec ftm.blockNumber>
I am not getting lastest block <37676547> could you please help me to find out the issues

**Reply by quan8** [reply, marked as answer]
When you see those lines "State heal in progress", it's not finished syncing yet.
You may wait till it shows "New block ...".

**Reply by HarishankarVj12** [reply]
How many days it will take to completed full Node

**Reply by quan8** [reply]
Depending on your machine specs. On a bare metal with NVME and good connection, it takes around 8 hours.

**Reply by HarishankarVj12** [reply]
But I have given High configuration IOPS 64000 for NVME stroage But it still syncing progress
